crates/router/src/compatibility/stripe/payment_intents.rs

```diff
@@ -78,12 +79,14 @@ pub async fn payment_intents_retrieve(
         connector: None,
         param: None,
         merchant_connector_details: None,
+        client_secret: query_payload.client_secret.clone(),
     };
 
-    let (auth_type, auth_flow) = match auth::get_auth_type_and_flow(req.headers()) {
```

Comment:
Are we moving completely to secret key model, or should we also support apikey format?